### PR TITLE
feat: refactor pages

### PR DESCRIPTION
**Pull request by vermasavita**
Below are the added pages:
- login page
- sign up page
- product listing page
- shopping cart page
- wishlist page

Preview: 
https://user-images.githubusercontent.com/82254618/161437120-53f2cedd-2239-4a58-9b23-febba0fa821f.mov

### CODE REVIEW

File: byBook/src/components/nav/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <form action="">
+          <input id="search" type="search" placeholder="Search" />
+          <label htmlFor="search" className="bx bx-search-alt-2"></label>
+        </form>
+      </div>
+
+      <div className="action-icon">
+        <ul className="dropdown">
+          <li className="btn">
+            <a className="link" href="#">
```
Comment (TejasShekar) [inline byBook/src/components/nav/Navbar.jsx:17]: replace `a` tags with `Link` tags

File: byBook/src/pages/Authentication/signup/signup.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/* .login-credential .confirm-password label, .login-credential .login-password label{
+    font-size: 1.5rem;
+    position: absolute;
+    top: 2.5rem;
+    right: 1.5rem;
+    color: var(--black-color);
+} */
```
Comment (TejasShekar) [inline byBook/src/pages/Authentication/signup/signup.css:7]: remove comments if not needed

**Review by TejasShekar — APPROVED** [review]
Looks good. Have added few suggestions. 
One general suggestion:
- Please add a preview page. Either use netlify which auto deploys previews for each PR you pull or make use of github pages.

File: byBook/src/App.js [hunk 3]
```diff
@@ -1,41 +1,19 @@
...
-            Contribute
-          </a>
-        </div>
-      </header>
+
+      <ProductListing/>
+      <CartListing/>
+      <Wishlist/>
+      <Login/>
+      <Signup/>
```
[the remark this answers is not in the document]
Comment (vermasavita) [reply byBook/src/App.js:16]: Sure.... after this PR. I'll add it.

**Review by AtulPant2704 — APPROVED** [review]

**Pull request merged**